firmware/hw_layer/drivers/can/can_hw.cpp

```diff
@@ -278,7 +278,8 @@ void initCan(void) {
 
 	// fire up threads, as necessary
 	if (CONFIG(canWriteEnabled)) {
-		canWrite.setPeriod(CONFIG(canSleepPeriodMs));
+		//remove user interaction to ensure we do have correct period timing
+		canWrite.setPeriod(CAN_CYCLE_PERIOD);
```

Comment:
you don't need this here if the period is set once at startup

Reply:
i can remove it, it stayed there since it was in the old code, but yes, it makes sense not to call it any more